tst/regression/CMakeLists.txt

```diff
@@ -110,3 +117,9 @@ foreach( index RANGE ${end_index})
   setup_test(${TEST_DIR} ${TEST_ARG})
   setup_test_mpi(${TEST_PROC} ${TEST_DIR} ${TEST_ARG})
 endforeach()
+
+# manually add label to performance regression test (to selectively enable/disable)
+set_tests_properties(regression_test:performance PROPERTIES LABELS "regression;mpi-no;perf-reg" )
+if( MPI_FOUND )
+  set_tests_properties(regression_mpi_test:performance PROPERTIES LABELS "regression;mpi-yes;perf-reg" )
+endif()
```

Comment:
This is fine as it is, but it points to an underlying problem with how I implemented the cmake script in this case. The above foreach loop and functions would ideally be flexible enough to allow for extra labels and other customization without having to introduce if branches like this, which makes the script less readable. I will create an issue to come up with a better design.